site/_posts/2017-01-30-keeping-up-with-the-conferences.md

```diff
@@ -0,0 +1,42 @@
+---
+title: "Last Week in ManageIQ: Keeping up with the conferences"
+author: Dávid Halász
+date: 2017-01-30 09:39:10 +01:00
```

Comment:
Didn't see this post in the front page, it was added however after removing the date portion here:
```
date: 2017-01-30
```